packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -147,11 +147,11 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
         setTarget(redirectUrl);
       }
     } catch (error) {
-      if (error.responseBody && error.responseBody.error.globalErrors) {
+      if (error?.responseBody && error?.responseBody?.error?.globalErrors) {
```

Comment:
I am sure that not all of these optionals are needed. Also this makes the `&&` redundant. Should probably be something more like

```suggestion
      if (error.responseBody?.error?.globalErrors) {
```